packages/promise-kit/src/promiseKit.js

```diff
@@ -49,19 +50,20 @@ export function makePromiseKit() {
   // Node.js adds the `domain` property which is not a standard
   // property on Promise. Because we do not know it to be ocap-safe,
   // we remove it.
-  if (p.domain) {
+  if ('domain' in p) {
     // deleting p.domain may break functionality. To retain current
     // functionality at the expense of safety, set unsafe to true.
     const unsafe = false;
     if (unsafe) {
-      const originalDomain = p.domain;
+      const originalDomain = /** @type {*} */ (p).domain;
```

Comment:
Is `*` an alias for `any`?

Reply:
Yup

Reply:
The convention we have in several of the packages is to use `any` when we really mean `any`, and reserve `*` for when we mean "I can't be bothered to type this right now, so I guess I'll punt until somebody cleans up after my mess."

Given that, I'd recommend:
```suggestion
      const originalDomain = /** @type {any} */ (p).domain;
```

Reply:
> The convention we have in several of the packages is to use any when we really mean `any`, and reserve `*` for when we mean "I can't be bothered to type this right now, so I guess I'll punt until somebody cleans up after my mess."

Cool. I didn't know that. Is it explained anywhere?